lib/graphiti/util/serializer_relationships.rb

```diff
@@ -49,10 +49,13 @@ def block
         data_proc_ref = data_proc
         self_ref = self
         validate_link! if eagerly_validate_links?
-
         proc do
           data { instance_eval(&data_proc_ref) }
 
+          # include relationship links for belongs_to relationships
+          # https://github.com/graphiti-api/graphiti/issues/167
+          linkage always: true if sideload_ref.type == :belongs_to
```

Comment:
This should do the trick for functionality. As far as code design goes, ideally the parent class wouldn't have to know about children. Maybe this could be an option that is overridden in the belongs_to subclass. We could also allow people to change the default value as well.

Reply:
yeah, could add an option. Based on what I described though, can you think of a reason someone wouldn't want this? It's still valid json-api, it's just more detailed json-api without a performance hit. 

As for placement: I put it here to let jsonapi-serializable do the heavy lifting but I still have no idea why this doesn't seem to work in the tests, do you?

Reply:
I agree might make sense to make this an option on `Sideload` generally, that is set for `Sideload::BelongsTo`:

```ruby
# override
belongs_to :post do
  linkage_always(false)
end

# opt-in has many
has_many :comments do
  linkage_always(true)
end
```

You can see code for something like `Sideload#assign_each` as a reference. I'm not sure about the specs without seeing a failing example.

Reply:
@jkeen, I can't see a good reason why someone might want to turn it off (though it's possible there is one), but I'm more interested in the case where you may want to turn it on for something else. This especially applies in the situation where your adapter isn't a relational database.